src/components/FullPage.js

```diff
@@ -64,6 +65,10 @@ class FullPage extends React.Component {
   }
 
   scrollToSlide(slide) {
+    if(this.props.onChange !== undefined) {
+      this.props.onChange(slide);
```

Comment:
This is closer to beforeChange, because scrolling takes time and this will run before any action. I think the perfect way to provide a useful hooks is to make two callbacks - beforeChange, afterChange.